client/state/data-layer/wpcom/sites/comments-tree/index.js

```diff
@@ -35,16 +35,22 @@ export const fetchCommentsTreeForSite = ( { dispatch }, action ) => {
 	);
 };
 
+const mapTree = ( tree, status, type ) => map( tree, comment => ( {
+	commentId: comment[ 0 ],
+	commentParentId: comment[ 2 ],
+	postId: comment[ 1 ],
+	status,
+	type,
+} ) );
```

Comment:
how about using some destructuring to remove the numerical indexing?

```js
map( tree, ( [ commentId, postId, commentParentId ] ) => {
	commentId,
	commentParentId,
	postId,
	status,
	type,
} )
```

Reply:
Oooh I like this!